CHANGELOG.md

```diff
@@ -1,22 +1,22 @@
 # Changelog
 
-All notable changes to Docker and Docker Compose resources for WSO2 Identity and Management version `6.0.x`
+All notable changes to Docker and Docker Compose resources for WSO2 Identity and Management version `6.1.x`
 in each resource release, will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
-## v6.0.0.1 - 2022-08-15
+## v6.1.0.1 - 2023-02-02
 
 ### Changed
 - Update Docker OS base image.
 
 ### Added
 
-- Docker resources of WSO2 Identity Server version `6.0.0` for Alpine and Ubuntu (refer to [issue](https://github.com/wso2/docker-is/issues/238))
+- Docker resources of WSO2 Identity Server version `6.1.0` for Alpine and Ubuntu (refer to [issue](https://github.com/wso2/docker-is/issues/238))
 
 ### Removed
 
 - JDK8 support
 
 For detailed information on the tasks carried out during this release, please see the GitHub milestone
-[v6.0.0.1](https://github.com/wso2/docker-is/milestone/23).
+[v6.1.0.1](https://github.com/wso2/docker-is/milestone/23).
```

Comment:
milestone should be 24

Reply:
Will update it. Can you create the milestone?

Reply:
created

Reply:
Addressed in https://github.com/wso2/docker-is/pull/366/commits/91275c755f25243bc73220b51b86c90e25515291